thorlcr/slave/slwatchdog.cpp

```diff
@@ -98,7 +98,9 @@ class CGraphProgressHandlerBase : public CSimpleInterface, implements ISlaveWatc
                 ForEachItemIn(g, activeGraphs) // NB: 1 for each slavesPerProcess
                 {
                     CGraphBase &graph = activeGraphs.item(g);
-                    graph.serializeStats(progressData);
+                    progressData.append((unsigned)graph.queryJobChannel().queryMyRank()-1);
+                    if (!graph.serializeStats(progressData))
```

Comment:
It used to serialize the slave # per graph, inc. child graphs.
Now it sends it once per subgraph, serialized here, not in graph object, avoiding it being serialized per child graph as before, which the master did not expect when deserializing it.